kubeconfig_test.go

```diff
@@ -93,3 +99,226 @@ func TestKubeConfig_getRESTConfigFromSecret(t *testing.T) {
 
 	}
 }
+
+func Test_KubeConfig_Unmarshal(t *testing.T) {
```

Comment:
Nitpick but unmarshal should be lower case to match the method. Same for marshal.